.travis.yml

```diff
@@ -21,8 +21,15 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+services:
+  - docker
+
+before_install:
+  - docker pull ibmcom/ibmjava:8-sdk
+
 script:
   - mvn test apache-rat:check clirr:check checkstyle:check findbugs:check javadoc:javadoc
+  - docker run -v `pwd`:/work ibmcom/ibmjava:8-sdk /bin/bash -c "apt-get update && cd work && apt-get install -y maven && mvn"
```

Comment:
Hi @britter @garydgregory @chtompki 

Flashed a though, can we use our own image for this ? we can build it from ```library/ibmjava:8-sdk```. Rather doing ```apt-get update``` and  ```apt-get install -y maven``` again and again on fly we can just build our own with below Dockerfile  on [Dockerhub](https://hub.docker.com), I also found we have [apache account](https://hub.docker.com/u/apache/) so it will be easy to publish it there. same can be reused by some other project in apache. _(that's why commons is made for anyway :blush: )_

```
from library/ibmjava:8-sdk

RUN apt-get update 
RUN apt-get install -y maven  
```